zebra-consensus/src/transaction/tests.rs

```diff
@@ -564,6 +564,83 @@ async fn mempool_request_with_past_lock_time_is_accepted() {
     );
 }
 
+#[tokio::test]
+async fn mempool_response_maturity_height_correct() {
```

Comment:
Can you add a test for the success case as well?

Nitpick: this test's name makes it seem like it is testing the success case, and it has no documentation.

Reply:
It actually was a test for the success case when it was returning `maturity_height: Some(_)` instead of an error, now the success case is covered by the test right above it.

Renamed/updated for the failure case and documented in https://github.com/ZcashFoundation/zebra/pull/6510/commits/e6c6c0f33593095568ffef531379d445f2baa290.